docs/markdown/Writing Plugins/rules-api/rules-api-and-target-api.md

```diff
@@ -317,7 +317,7 @@ async def demo(...) -> Foo:
     )
 ```
 
-If the provided `SourcesField` object is already a subclass of one of the `for_sources_types`—or it can be generated into one of those types—then the sources will be hydrated; otherwise, you'll get back a `HydratedSources` object with an empty snapshot and the field `sources_type=None`.
+If the provided `SourcesField` object is already a subclass of one of the `for_sources_types`—or it can be generated into one of those types—then the sources will be hydrated; otherwise, you'll get back a `HydratedSources` object with an empty snapshot and the field `sources_type=None`.
```

Comment:
GitHub doesn't show the diff properly, but I've removed a glyph at the end of the line that was there, see the image from the [readme docs](https://www.pantsbuild.org/docs/rules-api-and-target-api).

![image](https://user-images.githubusercontent.com/50622389/205489560-cda3c28e-7e55-46e2-8b2e-ff01d82be50d.png)